Resources/ArmTemplates/template.json

```diff
@@ -172,6 +191,33 @@
                 "[variables('AppServiceAppInsightsName')]"
             ]
         },
+        {
+            "name": "editor-certificate",
```

Comment:
q: Does the app service deployment need to be dependant on this?

Reply:
Yes, added dependency